core/trino-main/src/main/java/io/trino/security/AccessControl.java

```diff
@@ -222,7 +222,7 @@
     void checkCanShowColumns(SecurityContext context, CatalogSchemaTableName table);
 
     /**
-     * Filter the list of columns to those visible to the identity.
+     * Filter the list of columns to those visible to the identity when executing SHOW COLUMNS, DESCRIBE, etc.
```

Comment:
Revert especially given that it's also used in SELECT (conditionally).

(same in other interfaces)